### ISSUE TITLE
HY000 (11): database disk image is malformed

### ISSUE THREAD

**Issue by djrranga** [SimonOrJ/CoreProtect-Lookup-Web-Interface#35]
Occurs when making a query on the web interface. Database is SQLite and can be queried using the SQLite CLI. However, submitting any query to CoLWI returns this error.

**Comment by chuushi** [comment]
Hi Keno, does the `.db` file open properly for CoreProtect?

**Comment by djrranga** [comment]
Yep, CoreProtect is apparently able to read and write to the database file. I can also query it manually from the CLI.
